lib/Models/WebProcessingServiceCatalogFunction.js

```diff
@@ -53,6 +56,13 @@ function WebProcessingServiceCatalogFunction(terria) {
      */
     this.identifier = undefined;
 
+    /**
+     * Gets or sets whether to use key value pairs (KVP) embedded in Execute URL, or whether to make a POST request
+     * with XML data.
+     * @type {Boolean}
+     */
+    this.useKvp = undefined;
```

Comment:
Well the spec says POST is required, so theoretically such people shouldn't exist.  `GetCapabilities` also advertises which methods are supported, so we could check that to be super thorough:

![image](https://cloud.githubusercontent.com/assets/924374/22535532/d4fa118a-e94f-11e6-882f-79edae1230a7.png)